### PR TITLE
GlusterFS: Add GlusterFS hosts to openshift-hosted/config.yml playbook

### PR DESCRIPTION
**Pull request by jarrpa**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1575410

Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

File: playbooks/openshift-hosted/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 - import_playbook: ../init/main.yml
   vars:
-    l_init_fact_hosts: "oo_masters_to_config"
+    l_init_fact_hosts: "oo_masters_to_config:glusterfs_registry:glusterfs"
```
Comment (michaelgugino) [inline playbooks/openshift-hosted/config.yml]: @jarrpa are there any corresponding oo_ groups for these gluster hosts, or we don't have any for those?  I know in some other files we use oo_gluster_to_config, which would be helpful if we can stay consistent.

Comment (jarrpa) [reply playbooks/openshift-hosted/config.yml]: Sorry for the delay, this fell off my radar. Totally right, updating.

**Pull request merged**